tcases-lib/src/main/java/org/cornutum/tcases/Tcases.java

```diff
@@ -1185,141 +847,58 @@ else if( options.getTransformType() == Options.TransformType.HTML)
       }
     }
 
-  /**
-   * Returns test case definitions for the given system input definition, using the given generator set and
-   * base test definitions. If <CODE>genDef</CODE> is null, the default generator is used.
-   * If <CODE>baseDef</CODE> is null, no base test definitions are used.
-   */
-  public static SystemTestDef getTests( SystemInputDef inputDef, IGeneratorSet genDef, SystemTestDef baseDef, Options options)
-    {
-    if( genDef == null)
+    /*
+     * @deprecated Use TcasesCore.getTests
+     */
+    @Deprecated()
```

Comment:
all these deprecated methods below are of course not strictly required, just provided as a courtesy to clients